docs/defaulting.md

```diff
@@ -0,0 +1,249 @@
+# Ahead of Time libvirt Defaulting on the Cluster/API Level
+
+This Proposal fills a gap between KubeVirt as Runtime and Higher level
+management, like proposed in
+[docs/config-and-instances.md](docs/config-and-instances.md). It lays out a
+design which overcomes the necessity for asynchronous specification updates,
+which has it's origin in the fact that a lot of the defaulting is done on the
+host by libvirt, so after scheduling.
+
+With the proposal here, which will provide a synchronous way of fully
+populating a VM specification, we come much closer to the meaning of a Pod
+specification in Kubernetes.
+
+Note that a fully populated VM specification in the context of this proposal, is
+a specification which has filled out all fields, which influence how the
+emulated machine looks like for the Guest OS. For instance libvirt network
```

Comment:
You should probably reference the [`vm-configuration`](https://github.com/kubevirt/kubevirt/blob/master/docs/vm-configuration.md) proposal here which draws the clear line between backend- and frontend configuration of the specification. Thus the ones affecting the guess and the ones affecting the backend.

For the sake of an aligned language it's then easy to say that this defaulting would be limited to the guest facing, thus frontend configuration.